Source/Core/Core/PowerPC/Gekko.h

```diff
@@ -528,6 +528,17 @@ union UReg_FPSCR
     FI = 0;
     FR = 0;
   }
+
+  void SetException(u32 exc_mask)
```

Comment:
This abbreviation feels weird, any thoughts on naming it `exception_mask` instead?

Reply:
I can do that. I only renamed it in the first place because it shadows the static member variable `mask`, which the Windows build didn't like … (and the default CMake build completely ignores, grumble, grumble …)